### PR TITLE
fix: bulleted lists and interactive elements

### PR DESCRIPTION
**Pull request by BisuGit**
## Description
Interactive elements used to break the bulleted list structure. Added fix to allow both to work together. 

## DevQA
Run Example Project and ensure that the bulleted list with interactive elements appear and are functioning. 

### DevQA Prep

- Run `pod install`
- Test `RichTextView` to ensure that it passes (unit and UI tests)
- Navigate to the `Example` project in the root `Example` folder and run the app
- Ensure that everything in the `Example` app looks visually correct

### CODE REVIEW

**Comment by BisuGit** [comment]
Not sure why UI test images needed to be update, but DevQA'ing seems to make no difference on what RTV actually displays

**Comment by tophat-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">
<p>If anything other than <code>RichTextView</code> appears in the code coverage section please add it<br />
to the <code>exclude_targets</code> property of the <code>xcov.report()</code> function in the Dangerfile</p>
</td>
    </tr>
  </tbody>
</table>

## Current coverage for RichTextView.framework is `86.12%`
Files changed | - | - 
--- | --- | ---
RichTextParser.swift | `89.81%` | :white_check_mark:
StringExtension.swift | `97.28%` | :white_check_mark:

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Source/Text Parsing/RichTextParser.swift [hunk 1]
```diff
@@ -74,50 +77,55 @@ class RichTextParser {
...
         )
-        let strippedInputAsAttributedString = strippedInputWithSpecialDataTypesHandled.output
-        var allParsingErrors = strippedInputWithSpecialDataTypesHandled.errors
-        let rangeOfStrippedInputAttributedString = NSRange(location: 0, length: strippedInputAsAttributedString.length)
+        let parsingErrors = strippedInputWithSpecialDataTypesHandled.errors
         let outputAttributedStringToReturn = NSMutableAttributedString(attributedString: strippedInputWithSpecialDataTypesHandled.output)
+        let HTMLAndMarkdownParsedString = self.parseHTMLAndMarkdown(withAttributedString: outputAttributedStringToReturn, parsingErrors: parsingErrors)
+        let mutableAttributedInput = HTMLAndMarkdownParsedString.0
+        let allParsingErrors = HTMLAndMarkdownParsedString.1
+        let parsedStringWithAttributes = self.getAttributesIfNessasary(
```
Comment (maicon-brauwers) [inline Source/Text Parsing/RichTextParser.swift]: small typo on Necessary

Comment (BisuGit) [reply Source/Text Parsing/RichTextParser.swift]: Good Catch, Thanks!

**Pull request merged**